app/views/users/index.html.slim

```diff
@@ -1,16 +1,19 @@
+= render 'layouts/navigation'
+
 - @users.each do |user| 
   table
-    tr
-      th Name
-      th Email
-      th Actions
-    tr
-      td  
-        = user.name
-      td
-        = user.email
-      td
-        = link_to "Add Friend", users_path(user), method: :post
+    - if user != current_user
```

Comment:
Is this still needed? I thought this was handled in the query on the controller?